store/tikv/2pc.go

```diff
@@ -224,9 +224,19 @@ func (c *twoPhaseCommitter) doActionOnKeys(bo *Backoffer, action twoPhaseCommitA
 	if action == actionCommit {
 		// Commit secondary batches in background goroutine to reduce latency.
 		twoPhaseCommitGP.Go(func() {
+			start := time.Now()
+			secondaryLockCleanupWorkerGauge.Inc()
+			defer func() {
+				secondaryLockCleanupWorkerGauge.Dec()
+				secondaryLockCleanupHistogram.WithLabelValues("commit").Observe(time.Since(start).Seconds())
+			}()
+
 			e := c.doActionOnBatches(bo, action, batches)
 			if e != nil {
 				log.Debugf("2PC async doActionOnBatches %s err: %v", action, e)
+				secondaryLockCleanupTaskCounter.WithLabelValues("commit", "fail").Inc()
+			} else {
+				secondaryLockCleanupTaskCounter.WithLabelValues("commit", "ok").Inc()
```

Comment:
I think we just need record on fail condition.

Reply:
Maybe we want to know the percent of the failure, sometimes it's more useful than a single number.

Reply:
ok, though it has a global lock, I think it is better to remove.